src/main/java/atreides/task/Deadline.java

```diff
@@ -0,0 +1,33 @@
+package atreides.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
Should this variable be named more descriptively (such deadline, or timeBy), just to make its purpose clear?